docs/docs-content/clusters/edge/local-ui/cluster-management/update-cluster.md

```diff
@@ -0,0 +1,71 @@
+---
+sidebar_label: "Update Local Cluster"
+title: "Update Local Cluster"
+description: "Instructions for updating a locally manageg cluster in Edge Host Management Console."
+hide_table_of_contents: false
+sidebar_position: 40
+tags: ["edge"]
+---
+
+You can update existing local clusters from the local UI by uploading a new cluster profile version. This guide explains
+how to update an existing cluster in local UI.
+
+:::preview
+
+:::
+
+## Prerequisites
+
+- Network access to the Edge device’s IP and port where the local UI is exposed. The default port is 5080.
+
+- Credentials to log into the local UI. Any OS user can be used to log in to the local UI.
+
+- A local cluster created in local UI. For more information, refer to [Create Local Cluster](create-cluster.md).
+
+## Update Local Cluster
+
+1. Log into the local UI by visiting the 5080 port of your Edge device's IP address or domain name. For more
+   information, refer to [Access Local UI](../host-management/access-console.md).
+
+2. From the left **Main Menu**, select **Cluster**, and then select the **Configuration** tab.
+
+3. In the upper-right corner, select **Upload Configuration**.
+
+4. In the **Upload Configuration** wizard, browse or drag and drop the new cluster profile version **.tgz** file. Once
+   the upload finishes, select **Review Changes**.
+
+   To learn more about how to export a cluster profile and import it during this step, refer to
+   [Export Cluster Definition](./export-cluster-definition.md).
+
+5. Local UI displays the **Review Changes** modal, where you can review the update summary and verify all incoming
+   changes to the cluster profile variables. Your current configuration is displayed on the left, and the incoming
+   changes are displayed on the right.
+
+   For more information on cluster profile variables, refer to
+   [Define and Manage Profile Variables](../../../../profiles/cluster-profiles/create-cluster-profiles/define-profile-variables.md).
```

Comment:
when there are this many `..` i would just use absolute path. This way if anything moves this link won't bother you. No need to change it now but just something i do to make my life easier.

Reply:
@lenny we can't use absolute paths, we have to use relative paths otherwise it breaks versioning.

https://docusaurus.io/docs/markdown-features/links

Reply:
GitHub did me dirty 😓